admin/config/locales/products.en.yml

```diff
@@ -0,0 +1,9 @@
+en:
```

Comment:
@elia, I just saw there's a problem with these new translations as they interfere with the ones in `en.yml`  to generate the HTML title:

```html
<title>Sample Store - {:destroy=&gt;{:success=&gt;&quot;Products were successfully removed.&quot;}, :discontinue=&gt;{:success=&gt;&quot;Products were successfully discontinued.&quot;}, :activate=&gt;{:success=&gt;&quot;Products were successfully activated.&quot;}}</title>
```

Reply:
Fixed in https://github.com/solidusio/solidus/pull/5259